fe/fe-core/src/main/java/com/starrocks/connector/hive/HiveMetastoreApiConverter.java

```diff
@@ -168,10 +168,14 @@ public static HudiTable toHudiTable(Table table, String catalogName) {
 
     public static Partition toPartition(StorageDescriptor sd, Map<String, String> params) {
         requireNonNull(sd, "StorageDescriptor is null");
+        RemoteFileInputFormat fileFormat = toRemoteFileInputFormat(sd.getInputFormat());
+        if (fileFormat == null) {
```

Comment:
It also work for unpartitioned table?

Reply:
yes, both partitioned and unpartitioned table.